src/main/java/org/sonarlint/intellij/ui/SonarLintWalkthroughToolWindow.java

```diff
@@ -0,0 +1,482 @@
+
+    var welcomePage = new JPanel(new BorderLayout());
+    var learnAsYouCodePage = new JPanel(new BorderLayout());
+    var connectWithYourTeamPage = new JPanel(new BorderLayout());
+    var reachOutToUsPage = new JPanel(new BorderLayout());
+
+    var icon = new ImageIcon(Objects.requireNonNull(getClass().getResource("/images/sonarqube-for-ide-mark.png")));
+    var welcomeImageLabel = new JLabel(icon);
+    var learnAsYouCodeImageLabel = new JLabel(icon);
+    var connectWithYourTeamImageLabel = new JLabel(icon);
+    var reachOutToUsImageLabel = new JLabel(icon);
+
+    var welcomeStepLabel = new JLabel("Step 1/4", SwingConstants.LEFT);
+    welcomeStepLabel.setFont(new Font(FONT, Font.PLAIN, 14));
+
+    var titleLabel = new JLabel("Get started", SwingConstants.LEFT);
+    titleLabel.setFont(new Font(FONT, Font.BOLD, 16));
+    var welcomePageText = createWelcomePageText(font, project);
+
+    var welcomePageScrollPane = new JScrollPane(welcomePageText);
+    welcomePageScrollPane.setBorder(null);
+    welcomePageScrollPane.setPreferredSize(new Dimension(70, 100));
+
+    var welcomePageNextButton = new JButton("Next: Learn as you code");
+
+    var welcomePageNextButtonPanel = new JPanel(new FlowLayout(FlowLayout.RIGHT));
+    welcomePageNextButtonPanel.add(welcomePageNextButton);
+
+    createWelcomePageLayout(welcomeStepLabel, titleLabel, welcomePageScrollPane, welcomePageNextButtonPanel, welcomePage,
+      welcomeImageLabel);
+
+    var learnAsYouCodeStepLabel = new JLabel("Step 2/4", SwingConstants.LEFT);
+    learnAsYouCodeStepLabel.setFont(new Font(FONT, Font.PLAIN, 14));
+
+    var learnAsYouCodePageLabel = new JLabel("Learn as you code");
+    learnAsYouCodePageLabel.setFont(new Font(FONT, Font.BOLD, 16));
+
+    var learnAsYouCodeText = createLearnAsYouCodePageText(font, project);
+
+    var learnAsYouCodeScrollPane = new JScrollPane(learnAsYouCodeText);
+    learnAsYouCodeScrollPane.setBorder(null);
+    learnAsYouCodeScrollPane.setPreferredSize(new Dimension(70, 100));
+
+    var learnAsYouCodePageBackButton = new JButton(PREVIOUS);
+    var learnAsYouCodePageNextButton = new JButton("Next: Connect with your team");
+
+    var learnAsYouCodePageBackButtonPanel = new JPanel(new FlowLayout(FlowLayout.LEFT));
+    learnAsYouCodePageBackButtonPanel.add(learnAsYouCodePageBackButton);
+
+    var learnAsYouCodePageNextButtonPanel = new JPanel(new FlowLayout(FlowLayout.RIGHT));
+    learnAsYouCodePageNextButtonPanel.add(learnAsYouCodePageNextButton);
+
+    createLearnAsYouCodePageLayout(learnAsYouCodeStepLabel, learnAsYouCodePageLabel, learnAsYouCodeScrollPane,
+      learnAsYouCodePageBackButtonPanel, learnAsYouCodePageNextButtonPanel, learnAsYouCodePage, learnAsYouCodeImageLabel);
+
+    var connectWithYourTeamStepLabel = new JLabel("Step 3/4", SwingConstants.LEFT);
+    connectWithYourTeamStepLabel.setFont(new Font(FONT, Font.PLAIN, 14));
+    var connectWithYourTeamLabel = new JLabel("Connect with your team");
```

Comment:
Page titles and button labels should be in Title Case, I think.
```suggestion
    var connectWithYourTeamLabel = new JLabel("Connect with Your Team");
```

Reply:
This was recommended by Joseph